### DISCUSSION TITLE
V201 Initialization retry failed (The following sensor provider(s) failed initially: ADLXWrapper...)

### DISCUSSION THREAD

**Discussion by JanusTirdel** [Rem0o/FanControl.Releases#2720] [Q&A] [answered]
Any ideas? Started right after updating to V201

**Comment by Rem0o** [comment]
Retry mechanism was broken in the last few versions.   So this is just the error that was most likely supposed to happen.   Do you have an AMD video card?   Any log.txt?

**Reply by JanusTirdel** [reply]
I have integrated AMD GPU which I haven't disabled yet. 

9/9/2024 8:28:13 PM: Initialization failed. Retry count (2) exceeded.
9.9.2024 20.36.07: Initialization failed. Retry count (2) exceeded.

This is all that's in the log

I have Asrock B650 Pro RS and Ryzen 7 7800x3d

**Reply by Rem0o** [reply]
If you just hit CTRL+R (refresh), what error do you get in the logs?

**Reply by JanusTirdel** [reply]
Nothing. The app crashes and then if I open again I get the error in topic. When I restart I get the same error as above in the logs. Would clean install help and disabling the AMD GPU detection in the initial setup?

**Reply by Rem0o** [reply, marked as answer]
If you ain't got a dedicated AMD GPU, yes you should disable ADLX.

**Reply by JanusTirdel** [reply]
There we go. User error. Thank you. Will donate tomorrow. Now time for bed :)

**Reply by vonbalzer** [reply]
Reinstalled Windows 11.
I installed the program FanControl (v. 202), but it closes immediately after opening in the same way.
Refers to the ADLXWrapper
You wrote above that you need to disable ADLX - how do I do this?

**Reply by DanSparta** [reply]
> Reinstalled Windows 11. I installed the program FanControl (v. 202), but it closes immediately after opening in the same way. Refers to the ADLXWrapper You wrote above that you need to disable ADLX - how do I do this?

Second this, Not sure how to disable ADLX and googles not really helping me find anything

**Reply by DanSparta** [reply]
> Reinstalled Windows 11. I installed the program FanControl (v. 202), but it closes immediately after opening in the same way. Refers to the ADLXWrapper You wrote above that you need to disable ADLX - how do I do this?

Ok figured out a workaround, if you go into the installation folder, delete the ADLX Wrapper file then you can run FanControl enough to get to the setup popup where you can turn the ADLX option off

**Reply by vonbalzer** [reply]
Thank you very much! It helped! :)

**Comment by E11i0t09** [comment]
i'm on version V203 with the supposed ADXL fix, with a dedicated AMD Asus X6900XT OC AIO until recently the app worked like a charm even and finaly with the Asus AIO cooled card. But something went wrong, clicking the message ok makes the app run as usual with all controls able to use but with every start up i get this error. I tried reinstalling and checked the plugin?
![Screenshot_2890](https://github.com/user-attachments/assets/43395c2d-746a-4438-83e6-f1d75e0d5b1d)
![Screenshot_2891](https://github.com/user-attachments/assets/839ef070-545c-4450-b356-8ffd9e56424e)

**Reply by Rem0o** [reply]
The ADLX plugin is deprecated. Delete it and use the built-in ADLX.
